src/buildings/nobBaseWarehouse.cpp

```diff
@@ -1408,24 +1408,28 @@ bool nobBaseWarehouse::IsDependentFigure(noFigure* fig) const
 /// Available goods of a specific type that can be used for trading
 unsigned nobBaseWarehouse::GetAvailableWaresForTrading(const GoodType gt) const
 {
-    // We need a helper as leader
-    if(!inventory[JOB_HELPER])
+    // We need a scout as leader or a bow to recruit
+    bool noScoutAvailable = !inventory[JOB_SCOUT];
+    if (noScoutAvailable && !inventory[GD_BOW])
```

Comment:
Actually a bow and a helper and you should use `CanRecruit` instead

Reply:
actually you are right! i will add this.

Other question: 
Do you have access to the lobby server? For me it seems that the lobby is down.

Reply:
> Do you have access to the lobby server? For me it seems that the lobby is down.

thats fixed, gitlab upgrade changed some network config stuff